README.md

```diff
@@ -56,9 +56,7 @@ between these "modes" does come at a runtime cost.
 ### Building
```

Comment:
Thanks for mentioning the test failure! @mavangur, I know you've run into some issues before when converting between winrt chars and wchar_t. Could updating the SDK have potentially caused an issue in that conversion?

Reply:
HI Ben, you are right. I made some changes to the platform related to UiaChar. Right now we are using all winrt types in the RemoteOperations and the valid winrt type for wchar is char16_t.

 The Windows build environment defaults to using /Zc:wchar_t- for compatibility with early versions of the C++ compiler.
 Under this setting, the native C++ type wchar_t is not defined. Instead, wchar_t is a typedef for unsigned short.
 So while boxing the value, explicitly typecast it to char16_t as it is the valid winrt (16 bit) character type.

Since we are boxing with char16_t, we need to unbox using char16_t. But in UiaOpeartionAbstraction.cpp, we are unboxing with uint16_t which should be changed to char16_t. That should fix this issue.